opcua/gopcua/browser.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/mainflux/mainflux/opcua"
 )
 
-const maxChildrens = 7
+const maxChildrens = 5
```

Comment:
Same here - why 5, some comment please

Reply:
cant this be in env var

Reply:
We will remove this and change the architecture. No need for now